src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php

```diff
@@ -141,8 +142,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
+            $isStarted = WebServer::STARTED === $server->start($config, $input->getOption('pidfile'));
+            if ($isStarted && 'json' === $input->getOption('format')) {
+                printf(json_encode(array('hostname' => $config->getHostname(), 'port' => $config->getPort())));
+            }
 
-            if (WebServer::STARTED === $server->start($config, $input->getOption('pidfile'))) {
+            if ($isStarted && 'txt' === $input->getOption('format')) {
```

Comment:
It should default to txt. With the current implementation, if the user do a typo, the server will not output anything.

Reply:
You are right.